DeploymentExperiences.md

```diff
@@ -0,0 +1,4 @@
+I've done all the steps, but wasn't able to successfully deploy my django app to heroku on time.
+The most difficult parts of the process were figuring out the right settings for database configuration and environment variables. Documentation was pretty difficult to understand and was not as comprehensive as I needed it to be.
+
+P.S. After a good 2 hour nap, I finally realized what went wrong with my first deployment attempt. The problem was that I moved Pipfile and Procfile into the django project folder, but forgot to git stage the changes. So, the git did not remove these files after my original git push to heroku.
```

Comment:
Amazing that You manage to solve the issue. It is often very good to take a break and let mind rest.